manual_correction.py

```diff
@@ -180,112 +324,114 @@ def main():
     args = parser.parse_args()
 
     # Logging level
+    # TODO: how is this actually used?
     if args.verbose:
         coloredlogs.install(fmt='%(message)s', level='DEBUG')
     else:
         coloredlogs.install(fmt='%(message)s', level='INFO')
 
-    # check if input yml file exists
-    if os.path.isfile(args.config):
-        fname_yml = args.config
-    else:
-        sys.exit("ERROR: Input yml file {} does not exist or path is wrong.".format(args.config))
-
-    # fetch input yml file as dict
-    with open(fname_yml, 'r') as stream:
-        try:
-            dict_yml = yaml.safe_load(stream)
-        except yaml.YAMLError as exc:
-            print(exc)
+    # Fetch configuration from YAML file
+    dict_yml = utils.fetch_yaml_config(args.config)
 
     # Curate dict_yml to only have filenames instead of absolute path
     dict_yml = utils.curate_dict_yml(dict_yml)
 
-    # check for missing files before starting the whole process
-    utils.check_files_exist(dict_yml, args.path_in)
+    # Check for missing files before starting the whole process
+    if not args.add_seg_only:
+        utils.check_files_exist(dict_yml, utils.get_full_path(args.path_in))
+
+    suffix_dict = {
+        'FILES_SEG': args.suffix_files_seg,         # e.g., _seg or _label-SC_mask
+        'FILES_GMSEG': args.suffix_files_gmseg,     # e.g., _gmseg or _label-GM_mask
+        'FILES_LESION': args.suffix_files_lesion,     # e.g., _lesion
+        'FILES_LABEL': args.suffix_files_label,     # e.g., _labels or _labels-disc
+        'FILES_PMJ': '_pmj'
```

Comment:
same as in package for corrrection, we could add `label-pmj`

Reply:
Added in https://github.com/spinalcordtoolbox/manual-correction/pull/2/commits/b08a72f4a3e880cc5d177514dc67c7c05b496985